modules/apps/object/object-admin-rest-api/src/main/java/com/liferay/object/admin/rest/dto/v1_0/util/ObjectFieldUtil.java

```diff
@@ -42,6 +49,22 @@ public static JSONObject toJSONObject(ObjectField objectField) {
 			"indexedLanguageId", objectField.getIndexedLanguageId()
 		).put(
 			"label", objectField.getLabelMap()
+		).put(
+			"listTypeDefinitionExternalReferenceCode",
+			() -> {
+				if (!StringUtil.equals(
+						objectField.getBusinessType(),
+						ObjectFieldConstants.BUSINESS_TYPE_PICKLIST)) {
+
+					return null;
+				}
+
+				ListTypeDefinition listTypeDefinition =
+					listTypeDefinitionService.getListTypeDefinition(
+						objectField.getListTypeDefinitionId());
+
+				return listTypeDefinition.getExternalReferenceCode();
```

Comment:
@murilostodolni should we check if `listTypeDefinition` is null?

Reply:
Hi @carolmariaabb in this case i think that not necessary. Because is toDTO, is mandatory has a listTypeDefinitionId in database. What do you think?